index.html

```diff
@@ -246,6 +248,68 @@ <h4>Verifiable Data Registries</h4>
         </section>
       </section>
     </section>
+
+    <section>
+      <h3>Referencing Other Credentials</h3>
+      <p>
+Usage of <a>verifiable credentials</a> will often require referencing other
+credentials, embedding or attaching multiple credentials, or otherwise binding
+them together.
+      </p>
+
+      <section>
+        <h4>Referencing Credentials Without Integrity Protection</h4>
+        <p>
+The simplest way for a <a>credential</a> to reference another external
+credential is to link to it, either explicitly by using a URI, or implicitly by
+providing a well-known ID (for example, a credential modeling an internal
+company Invoice may refer to its parent Purchase Order credential simply by the
+PO Number, relevant only within the context of this specific enterprise).
+        </p>
+
+        <p>
+This method of linking to an external credential without using an integrity
+protection mechanism may be acceptable in rare use cases, such as when both
+credentials are issued by the same entity, the verifier has a high level
+of trust and confidence in the issuer's security and auditing mechanisms, and
+the value of the credentials is sufficiently low. However, this is not
+recommended, and goes against the philosophy of <a>verifiable credentials</a>.
+Although the credential that contains the reference may be integrity protected
+itself (by a cryptographic signature or a similar proof mechanism), the verifier
+has no way of knowing that the external credential being linked to has not been
+tampered with, unless the link itself has a content integrity protection
+mechanism built into it.
+        </p>
+      </section>
+
+      <section>
+        <h4>Referencing Credentials With Integrity Protection</h4>
+        <p>
+The recommended way of referencing an external credential from within a
+<a>verifiable credential</a> is to use a linking mechanism that
+cryptographically binds the contents of the target document to the URI itself.
+One way to accomplish this would be to use <a>hashlinks</a> or an equivalent
+URI scheme. Another mechanism would be to encode the full contents of the
```

Comment:
I dont see the point in encoding the full contents, especially if the issuer is the same for both credentials. I suggest deleting the section starting Another mechanism